pilot/pkg/serviceregistry/kube/controller/controller_test.go

```diff
@@ -1625,8 +1625,8 @@ func TestEndpoints_WorkloadInstances(t *testing.T) {
 		Name:      "ratings-2",
 		Namespace: "bookinfo-ratings",
 		Endpoint: &model.IstioEndpoint{
-			Labels:  labels.Instance{"app": "ratings"},
-			Address: "2.2.2.2",
+			Labels:    labels.Instance{"app": "ratings"},
+			Addresses: []string{"2.2.2.2"},
```

Comment:
TestEndpoints_WorkloadInstances() requires dual-stack tests

Reply:
done! add for `ratings-3`